packages/front-end/components/GeneralSettings/FeaturesSettings.tsx

```diff
@@ -99,7 +99,40 @@ export default function FeaturesSettings() {
             {...form.register("secureAttributeSalt")}
           />
         </div>
-
+        <div>
+          <label htmlFor="featureKeyExample">
+            Feature Key Example (Optional)
+          </label>
+          <Field
+            id="featureKeyExample"
+            {...form.register("featureKeyExample")}
+            placeholder="growth-20240531-newFeature"
```

Comment:
Remove this placeholder or replace with `my-feature`, the default example we use in the app today